a3/skeleton/phonebook_skeleton.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+PHONEBOOK_ENTRIES = ".python_phonebook_entries"
```

Comment:
they won't be able to find these without ls -la and will probably get confused, I'd not make them hidden files